src/app/pipes/display-by-delimiter.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { DisplayByDelimiter } from '@app/pipes/display-by-delimiter';
+
+describe('DisplayByDelimiter', () => {
+  const pipe = new DisplayByDelimiter();
+
+  it('create an instance', () => {
+    expect(pipe).toBeTruthy();
+  });
+
+  it('should return Yes if value is boolean', () => {
```

Comment:
`it should return the expected concatenation`. You also need 2 more tests.
- a test with concatenation and having no delimiter
- a test that takes empty string and does nothing